src/main.rs

```diff
@@ -302,8 +303,14 @@ async fn worker(
 
 #[actix_web::main]
 async fn main() -> Result<()> {
+    // Print --help information
+    let matches = ClapApp::new("keylime_agent")
+        .about("A Rust implementation of the Keylime agent")
```

Comment:
```suggestion
        .about("A Rust implementation of the Keylime agent")
        .override_usage("sudo RUST_LOG=keylime_agent=trace ./target/debug/keylime_agent")
```
This would help users know how it is actually meant to be run, since `keylime_agent` currently does not work for the Rust agent.

Reply:
Done! :pray: